modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -218,7 +218,8 @@
 			});
 
 			if( this.getConfig('layout') == 'below'){
-				this.updateBelowVideoCaptionContainer();
+                this.getPlayer().triggerHelper("removeTextSize");
```

Comment:
why bind this via event? you can know in advance you are in `below` mode and init the CVAA plugin accordingly: https://github.com/kaltura/mwEmbed/pull/3660/files#diff-322dfb3fd3c5e59a19ccacb340d0fc71L55

Reply:
@OrenMe made the changes.